### PR TITLE
chore: Included githubactions in the dependabot config

### PR DESCRIPTION
**Pull request by naveensrinivasan**
This should help with keeping the GitHub actions updated on new releases. This will also help with keeping it secure.

Dependabot helps in keeping the supply chain secure https://docs.github.com/en/code-security/dependabot

GitHub actions up to date https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool
Signed-off-by: naveen <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by Court72** [comment]
@EdPrice-MSFT 

Can you review the proposed changes? The commits should be moved into the private repo to benefit from validation as well.

*IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

**Comment by EdPrice-MSFT** [comment]
@ckittel or @josevarela - This one too? Thanks!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+      interval: "weekly"
```
Comment (ckittel) [inline .github/dependabot.yml:6]:
```suggestion
    interval: "weekly"
```

**Review by ckittel — CHANGES_REQUESTED** [review]

**Comment by ckittel** [comment]
I'm going to close this one in retrospect.  This repo isn't the place where we'd want to open this PR as we have a "backing repo" for this instead, these PRs would be noise in the public repo.  This repo is synced from a backend repo and is only here to support issues and PRs to adjust content.  Thank you though @naveensrinivasan!

**Pull request closed without merging**